.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Report a Bug within the product
+title: ''
+labels: 'Bug Fix'
+assignees: ''
+
+---
+
+<!--
+
+Have you read DDF's Code of Conduct? By filing an Issue, you are
+expected to comply with it, including treating everyone with respect:
+https://github.com/codice/ddf/blob/master/.github/CODE_OF_CONDUCT.md
+
+Do you want to ask a question? Are you looking for support? The DDF
+Developers group - https://groups.google.com/forum/#!forum/ddf-developers
+is the best place for getting support.
+
+-->
+
+### Description
+<!-- 
+Description of the issue
+Why is this change being proposed?
+Why should this be added to the codebase?
+-->
+
+### Steps to Reproduce
+<!--
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+-->
+
+#### Expected behavior :
+<!-- What you expect to happen -->
+
+#### Actual behavior:
+<!-- What actually happens -->
+
+#### Reproduces how often:
```

Comment:
✏️ I'm not sure if it needs its own section, but can we add something about potential workarounds?
Maybe that's best suited for the Description block.

Reply:
Good point :+1: I'm looking at the description within the bug template and it could go there. I don't think we need to have `why is this change being proposed` when it's a bug